src/fiatconnect/TransferStatusScreen.tsx

```diff
@@ -0,0 +1,203 @@
+import { StackScreenProps } from '@react-navigation/stack'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { ActivityIndicator, SafeAreaView, StyleSheet, Text, View } from 'react-native'
+import { fiatConnectTransferSelector } from 'src/fiatconnect/selectors'
+import { useSelector } from 'react-redux'
+import variables from 'src/styles/variables'
+import fontStyles from 'src/styles/fonts'
+import React from 'react'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import BackButton from 'src/components/BackButton'
+import TextButton from 'src/components/TextButton'
+import { emptyHeader } from 'src/navigator/Headers'
+import { Spacing } from 'src/styles/styles'
+import { useTranslation } from 'react-i18next'
+import { navigate, navigateBack, navigateHome } from 'src/navigator/NavigationService'
+import CheckmarkCircle from 'src/icons/CheckmarkCircle'
+import OpenLinkIcon from 'src/icons/OpenLinkIcon'
+import Touchable from 'src/components/Touchable'
+import networkConfig from 'src/web3/networkConfig'
+import colors from 'src/styles/colors'
+import { FiatConnectTransfer } from 'src/fiatconnect/slice'
+import FiatConnectQuote from 'src/fiatExchanges/quotes/FiatConnectQuote'
+
+type Props = StackScreenProps<StackParamList, Screens.FiatConnectTransferStatus>
+
+function onBack() {
+  // TODO: navigate to ReviewFetchScreen once #2699 is merged
+  navigateBack()
+}
+
+function FiatConnectWithdrawFailureSection() {
+  const { t } = useTranslation()
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>{t('fiatConnectStatusScreen.requestNotCompleted.title')}</Text>
+      <Text style={styles.description}>
+        {t('fiatConnectStatusScreen.requestNotCompleted.description')}
+      </Text>
+      <Button
+        style={styles.button}
+        testID="TryAgain"
+        onPress={onBack}
+        text={t('fiatConnectStatusScreen.tryAgain')}
+        type={BtnTypes.PRIMARY}
+        size={BtnSizes.MEDIUM}
+      />
+      <Button
+        style={styles.button}
+        testID="SupportContactLink"
+        onPress={() => {
+          navigate(Screens.SupportContact, {
+            prefilledText: t('fiatConnectStatusScreen.requestNotCompleted.contactSupportPrefill'),
+          })
+        }}
+        text={t('contactSupport')}
+        type={BtnTypes.SECONDARY}
+        size={BtnSizes.MEDIUM}
+      />
+    </View>
+  )
+}
+
+function FiatConnectWithdrawSuccessSection({
+  fiatConnectTransfer,
+  normalizedQuote,
+}: {
+  fiatConnectTransfer: FiatConnectTransfer
+  normalizedQuote: FiatConnectQuote
+}) {
+  const { t } = useTranslation()
+  const timeEstimation = normalizedQuote.getTimeEstimation()!
```

Comment:
Should there be a fallback if this is null?

Reply:
Currently this will always be populated, but yeah, a fallback is probably a good idea. Not sure what we should set it to though. Any thoughts?

Reply:
hmm maybe some fallback text without duration? Probably need Laura/Nitya's input. We could also address this as part of the TODO to make this dynamic. It probably also needs some update if settlement times are in the magnitude of hrs right?

Reply:
I left a TODO for now. I can spin up a ticket to make settlement times/durations dynamic as well.